chromadb/test/property/test_embeddings.py

```diff
@@ -134,12 +134,14 @@ def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID
                 if normalized_record_set["embeddings"]
                 else None,
             }
-            self.collection.add(**normalized_record_set)  # type: ignore[arg-type]
+            result = self.collection.add(**normalized_record_set)  # type: ignore[arg-type]
```

Comment:
We don't currently generate `RecordSet` without IDs. Do we want to do that in property tests? This might get more complicated if we want to allow some IDs to be `None`.

Reply:
What happens in the tests when we first do an `add` with IDs, then one without? The comparison here would fail I think.